docs/community/community-membership.md

```diff
@@ -0,0 +1,174 @@
+description: Community organisation and members responsibilities
+<!--- END of page meta data -->
+
+!!!note
+    This document is in progress
+
+This doc outlines the various responsibilities of contributor roles in
+this project.
+
+| Role | Responsibilities | Defined by |
+| -----| ---------------- | ---------- |
+| Everyone | none | anybody with a belly button
+| Member | everyone who contributes - code or otherwise | Orion GitHub org member
+| Approver | approve accepting contributions | write permissions on master
+| Project Manager | management of the project | PegaSys
+| Project Sponsor | contribute developer resources | PegaSys
+| Open Source Circle | OSS support | PegaSys
+| Project Evangelist | promote the project | PegaSys
+| Benevolent Dictator | decision tie-breaker | PegaSys
+
+## Everyone
+Any person from the public is able to access the code.  The standard permissions grant the ability to view the code, view open bugs, access the wiki, download binaries, view CI results and comment on pull requests.
+
+## New contributors
+
+[New contributors] should be welcomed to the community by existing members,
+helped with PR workflow, and directed to relevant documentation and
+communication channels.
+
+## Established community members
+
+Established community members are expected to demonstrate their adherence to the
+principles in this document, familiarity with project organization, roles,
+policies, procedures, conventions, etc., and technical and/or writing ability.
+Role-specific expectations, responsibilities, and requirements are enumerated
+below.
+
+## Member
+
+Members are continuously active contributors in the community.  They can have
+issues and PRs assigned to them.
+
+### Requirements
+
+- Enabled [two-factor authentication] on their GitHub account
+- Have made multiple contributions to the project or community.  Contribution may include, but is not limited to:
+    - Authoring or reviewing PRs on GitHub
+    - Filing or commenting on issues on GitHub
+    - Contributing to community discussions (e.g. meetings, Slack, email discussion forums, Stack Overflow)
+- Subscribed to [pantheon-dev@pegasys.tech]
```

Comment:
🤔is it really working ? I had no answer or error (yet) on the address

Reply:
Tagged you in on a thread